package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "world-counter",
+  "version": "0.0.0",
+  "license": "MIT",
+  "angular-cli": {},
+  "scripts": {
+    "start": "ng serve",
+    "lint": "tslint \"src/**/*.ts\"",
+    "test": "ng test",
+    "pree2e": "webdriver-manager update",
+    "e2e": "protractor"
+  },
+  "dependencies": {
+    "@angular/common": "2.4.1",
```

Comment:
@yannickglt Be a bit more optimistic ...
https://github.com/angular/angular/releases/tag/4.0.0-beta.1